hr_family/models/hr_children.py

```diff
@@ -21,6 +21,9 @@
 
 from openerp import models, fields
 
+GENDER_SELECTION = [('m', 'M'),
```

Comment:
Can you keep the same values as those defined for the field `gender` on `hr.employee`https://github.com/odoo/odoo/blob/8.0/addons/hr/hr.py#L197
```python
'gender': fields.selection([('male', 'Male'), ('female', 'Female')], 'Gender'),
```